### ISSUE TITLE
swirl : error in assignment for "Exploratory Data Analysis"

### ISSUE THREAD

**Issue by sachin1539** [swirldev/swirl#737]
I am getting error in assignment for "Exploratory Data Analysis"
I installed the Package ‘ggplot2’ correctly!
Package ‘jpeg’also  loaded correctly! 
I am exited from Swirl() with the message "Error in plot.new() : figure margins too large"
Need the solution to fix this.

**Comment by jldeutsch** [comment]
I don't think this is an issue with swirl. If you try to make a plot in RStudio and the plotting pane is too small it will throw this error.
